.github/workflows/operate-first-publish.yaml

```diff
@@ -0,0 +1,51 @@
+name: Update Operate First directory
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Update Operate First directory
+    runs-on: ubuntu-latest
+    steps:
+    
+      - name: Checkout Operate First repository
+        uses: actions/checkout@v2
+        with:
+          repository: eletonia/apps
+          path: operate-first-repo
+      - name: Checkout Fybrik repository
+        uses: actions/checkout@v2
+        with:
+          repository: eletonia/fybrik
+          path: fybrik-repo
+      - run: |
+          rm -r fybrik-repo/samples/operate-first/*/
+          rm fybrik-repo/samples/operate-first/*.yaml
+          cd fybrik-repo/hack/tools
+          ./install_yq.sh
+          ./install_helm.sh
+          cd ../../samples/operate-first
```

Comment:
you might want to use `pushd` and `popd` https://github.com/fybrik/fybrik/blob/master/.github/workflows/build.yml#L145